src/main/java/com/onarandombox/MultiversePortals/listeners/MVPPlayerListener.java

```diff
@@ -236,6 +236,13 @@ public void playerPortal(PlayerPortalEvent event) {
                     event.setCancelled(true);
                     return;
                 }
+
+                Location destLocation = portalDest.getLocation(event.getPlayer());
+                if (destLocation == null) {
+                    event.setCancelled(true);
```

Comment:
May be useful to put a some logging here, bcu it may point to some issue if destination is null. @nicegamer7 do you think the log level should be warning or fine?

```java
Logging.fine("Portal event is cancelled as destination is null!");
```

Reply:
I don't think `warning` would be good, since it would throw warnings if a player doesn't have a bed spawn, for example. I think `fine` would be good.

Reply:
I guess it would be good to have the same logging in the other file too.

Reply:
I think we can merge this, then have another for logging?

Reply:
Sure.